widgets/filters/src/widget.tsx

```diff
@@ -0,0 +1,37 @@
+import FilterActions from './FilterActions';
+import FiltersTable from './FiltersTable';
+import type { Filter, WidgetConfiguration } from './types';
+
+Stage.defineWidget<unknown, Stage.Types.PaginatedResponse<Filter>, WidgetConfiguration>({
+    id: 'filters',
+    name: Stage.i18n.t('widgets.filters.name'),
+    description: Stage.i18n.t('widgets.filters.description'),
+    initialWidth: 12,
+    initialHeight: 28,
+    color: 'olive',
+    isReact: true,
+    hasStyle: false,
+    hasReadme: false,
+    permission: Stage.GenericConfig.WIDGET_PERMISSION('filters'),
+    categories: [Stage.GenericConfig.CATEGORY.SYSTEM_RESOURCES],
+
+    initialConfiguration: [Stage.GenericConfig.POLLING_TIME_CONFIG(30), Stage.GenericConfig.PAGE_SIZE_CONFIG(10)],
+
+    fetchData(_widget, toolbox, params) {
+        return new FilterActions(toolbox).doList(params);
+    },
+
+    render(widget, data, _error, toolbox) {
+        if (Stage.Utils.isEmptyWidgetData(data)) {
+            const { Loading } = Stage.Basic;
+            return <Loading />;
```

Comment:
Optional:

We can do

```suggestion
            return <Stage.Basic.Loading />;
```

To save one line :slightly_smiling_face:

Reply:
Some time ago @qooban suggested to always use destructuring when using things from `Stage`, so I'm trying to align to this rule.

Reply:
Hmm... I thought that it was impossible before due to some ESLint rule that prevented it. I was not informed about such spoken rules :smile: 

@qooban do you think the rule still makes sense? Do you see benefits destructuring here?

Personally I prefer the one-line version :slightly_smiling_face:

Reply:
I'm OK with both approaches in this case, but to be consistent with other parts I would stay with destructuring. I don't see real benefits with having one line instead of two.

Reply:
I'm OK with both approaches as well.
Since this comment was marked as optional and there is no consistent opinion I'm leaving it as it is for now.